package.json

```diff
@@ -62,6 +61,7 @@
     "jest-cli": "^26.1.0",
     "jest-coverage-badges": "^1.1.2",
     "lint-staged": "^10.2.11",
+    "moment": "^2.28.0",
```

Comment:
Maybe slightly controversial but I figured there was no better way of testing this change than by keeping the test the same. So I've moved moment to be a dev dependency (so bundle size is minimal) but we still get to keep our good unit test. And other tests in the future can take advantage of moment.